templates/internet-of-things/shared/_links.html

```diff
@@ -0,0 +1,49 @@
+<section class="{{ strip_class }}">
+  <div class="row p-divider">
+    {% if current != "signage" %}
+    <div class="col-3 p-divider__block">
+      <h2 class="p-heading--3">Signage</h2>
+
+      <p>A small footprint and full OpenGL make Ubuntu Core the perfect platform for secure digital signs.</p>
+      <a href="/internet-of-things/smart-displays">Learn more&nbsp;&rsaquo;</a>
+    </div>
+    {% endif %}
+
+    {% if current != "gateways" %}
+    <div class="col-3 p-divider__block">
+      <h2 class="p-heading--3">Gateways</h2>
+
+      <p>Rich networking and comms make Ubuntu the perfect choice for your industrial gateways.</p>
+      <a href="/internet-of-things/gateways">Learn more&nbsp;&rsaquo;</a>
+    </div>
+    {% endif %}
+
+    {% if current != "automotive" %}
+    <div class="col-3 p-divider__block">
+      <h2 class="p-heading--3">Automotive</h2>
+
+
+      <p>Reinvent the automobile with Ubuntu, designed for critical embedded systems.</p>
+      <a href="/automotive">Learn more&nbsp;&rsaquo;</a>
+    </div>
+    {% endif %}
+
+    {% if current != "networking" %}
+    <div class="col-3 p-divider__block">
+      <h2 class="p-heading--3">Networking</h2>
+
+      <p>Build secure SmartNICs and operate them at scale on Ubuntu.</p>
+      <a href="/networking">Learn more&nbsp;&rsaquo;</a>
```

Comment:
This `href` should be `/internet-of-things/networking`

Reply:
:+1: